platform/windows/tts_windows.cpp

```diff
@@ -60,8 +60,8 @@ void __stdcall TTS_Windows::speech_event_callback(WPARAM wParam, LPARAM lParam)
 	}
 }
 
-void TTS_Windows::_update_tts() {
-	if (!is_speaking() && !paused && queue.size() > 0) {
+void TTS_Windows::procee_events() {
```

Comment:
I supposed this was meant to be
```suggestion
void TTS_Windows::process_events() {
```

Reply:
Fixed.